rdkit/Chem/SaltRemover.py

```diff
@@ -146,6 +196,13 @@ def StripMol(self, mol, dontRemoveEverything=False):
     2
 
     """
+    m, removed = self._StripMol(mol, dontRemoveEverything)
+    return m
+
+  def StripMolAndReturnRemoved(self, mol, dontRemoveEverything=False):
+    return self._StripMol(mol, dontRemoveEverything)
```

Comment:
I didn't understand the code at first and thought that you implemented the opposite behaviour. To make the first function more obvious, maybe use this instead:
```
mol, _ = self._StripMol(mol, dontRemoveEverything)
return mol
```
and 
```
mol, salts = self._StripMol(mol, dontRemoveEverything)
return mol, salts
```
This makes the intention clearer.